api/src/main/resources/workbench-api.yaml

```diff
@@ -0,0 +1,7370 @@
+      that all endpoints are protected: this annotation ensures that client
+      libraries know to send bearer tokens when calling
+    authorizationUrl: ''
+    tokenUrl: ''
+    type: oauth2
+    flow: accessCode
+schemes:
+  - https
+produces:
+  - application/json
+security:
+  - aou_oauth: []
+parameters:
+  userId:
+    in: path
+    name: userId
+    type: integer
+    format: int64
+    required: true
+  conceptSetId:
+    in: path
+    name: conceptSetId
+    type: integer
+    format: int64
+    required: true
+    description: Concept set ID
+  dataSetId:
+    in: path
+    name: dataSetId
+    type: integer
+    format: int64
+    required: true
+    description: Data set ID
+  workspaceNamespace:
+    in: path
+    name: workspaceNamespace
+    type: string
+    description: The Workspace namespace
+    required: true
+  workspaceId:
+    in: path
+    name: workspaceId
+    type: string
+    description: The Workspace ID (a.k.a. the workspace's Firecloud name)
+    required: true
+  cohortId:
+    in: path
+    name: cohortId
+    type: integer
+    format: int64
+    required: true
+    description: Cohort ID
+  cohortReviewId:
+    in: path
+    name: cohortReviewId
+    type: integer
+    format: int64
+    required: true
+    description: Cohort Review ID
+  cdrVersionId:
+    in: path
+    name: cdrVersionId
+    type: integer
+    format: int64
+    required: true
+    description: specifies which cdr version
+  participantId:
+    in: path
+    name: participantId
+    type: integer
+    format: int64
+    required: true
+    description: specifies which participant
+paths:
+  "/v1/status":
+    get:
+      tags:
+        - status
+      description: Returns the status of the various services and integrations
+      operationId: getStatus
+      security: []
+      responses:
+        200:
+          description: A map of system name to status.
+          schema:
+            "$ref": "#/definitions/StatusResponse"
+  "/v1/config":
+    get:
+      tags:
+        - config
+      description: Returns some server configuration data.
+      operationId: getConfig
+      security: []
+      responses:
+        200:
+          description: Configuration data
+          schema:
+            "$ref": "#/definitions/ConfigResponse"
+  "/v1/featured-workspace-config":
+    get:
+      tags:
+        - featured-workspaces-config
+      description: Returns the featured workspaces config
+      operationId: getFeaturedWorkspacesConfig
+      security: []
+      responses:
+        200:
+          description: List of featured workspaces
+          schema:
+            "$ref": "#/definitions/FeaturedWorkspacesConfigResponse"
+  "/v1/status-alert":
+    get:
+      tags:
+        - status-alert
+      description: Returns status alert information if any exists.
+      operationId: getStatusAlert
+      responses:
+        200:
+          description: 'A status alert information object.
+
```

Comment:
I don't agree this is an existing issue, seems more like a problem with the tool conversion - here is the previous code: https://github.com/all-of-us/workbench/blob/c0a2f5627aeb12f35369e70a3d65f688165dfad5/api/src/main/resources/workbench.yaml#L149-L150

That said, I don't feel strongly whether this gets fixed in a follow-up or here.

Reply:
Yeah, it's only existing w/r/t merged.yaml as a base. Ruby's parser seems to want to do that.

It should be a simple one-off string replacement (if this is the last time we use that Ruby Yaml client).